### PR TITLE
add Ivanti vuls from 2024-11-12

### PR DESCRIPTION
**Pull request by amanion-cisa**

### CODE REVIEW

File: csaf_files/IT/white/2024/va-24-317-01.json [hunk 1]
```diff
@@ -0,0 +1,898 @@
...
+        "category": "vendor",
+        "name": "Ivanti",
+        "branches": [
+          {
+            "category": "product_name",
+            "name": "Connect Secure",
+            "branches": [
+              {
+                "category": "product_version_range",
+                "name": "0 < 22.6R2",
```
Comment (justmurphy) [inline csaf_files/IT/white/2024/va-24-317-01.json]: This should be `"<22.6R2"`.

See documentation: https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#31232-branches-type---name-under-product-version-range

File: csaf_files/IT/white/2024/va-24-317-01.json [hunk 2]
```diff
@@ -0,0 +1,898 @@
...
+                "product": {
+                  "name": "Ivanti Connect Secure 0 < 22.6R2",
```
Comment (justmurphy) [inline csaf_files/IT/white/2024/va-24-317-01.json:93]: Change `"name"` field to reflect above change with version range.

File: csaf_files/IT/white/2024/va-24-317-01.json [hunk 3]
```diff
@@ -0,0 +1,898 @@
...
+                "category": "product_version",
+                "name": "22.6R2",
+                "product": {
+                  "name": "Ivanti Connect Secure 22.6R2",
+                  "product_id": "CSAFPID-0002"
+                }
+              },
+              {
+                "category": "product_version_range",
+                "name": "0 < 9.1R18.7",
```
Comment (justmurphy) [inline csaf_files/IT/white/2024/va-24-317-01.json]: This should be `"<9.1R18.7"`.

See documentation: https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#31232-branches-type---name-under-product-version-range

File: csaf_files/IT/white/2024/va-24-317-01.json [hunk 4]
```diff
@@ -0,0 +1,898 @@
...
+                "product": {
+                  "name": "Ivanti Connect Secure 0 < 9.1R18.7",
```
Comment (justmurphy) [inline csaf_files/IT/white/2024/va-24-317-01.json:109]: Change `"name"` field to reflect above change with version range.

File: csaf_files/IT/white/2024/va-24-317-01.json [hunk 5]
```diff
@@ -0,0 +1,898 @@
...
+                "category": "product_version",
+                "name": "9.1R18.7",
+                "product": {
+                  "name": "Ivanti Connect Secure 9.1R18.7",
+                  "product_id": "CSAFPID-0004"
+                }
+              },
+              {
+                "category": "product_version_range",
+                "name": "0 < 22.7R2.1",
```
Comment (justmurphy) [inline csaf_files/IT/white/2024/va-24-317-01.json]: This should be `"<22.7R2.1"`.

See documentation: https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#31232-branches-type---name-under-product-version-range

File: csaf_files/IT/white/2024/va-24-317-01.json [hunk 6]
```diff
@@ -0,0 +1,898 @@
...
+                "product": {
+                  "name": "Ivanti Connect Secure 0 < 22.7R2.1",
```
Comment (justmurphy) [inline csaf_files/IT/white/2024/va-24-317-01.json:125]: Change `"name"` field to reflect above change with version range.

File: csaf_files/IT/white/2024/va-24-317-01.json [hunk 7]
```diff
@@ -0,0 +1,898 @@
...
+                "category": "product_version",
+                "name": "22.7R2.1",
+                "product": {
+                  "name": "Ivanti Connect Secure 22.7R2.1",
+                  "product_id": "CSAFPID-0008"
+                }
+              },
+              {
+                "category": "product_version_range",
+                "name": "0 < 9.1R18.9",
```
Comment (justmurphy) [inline csaf_files/IT/white/2024/va-24-317-01.json]: This should be `"<9.1R18.9"`.

See documentation: https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#31232-branches-type---name-under-product-version-range

File: csaf_files/IT/white/2024/va-24-317-01.json [hunk 8]
```diff
@@ -0,0 +1,898 @@
...
+                "product": {
+                  "name": "Ivanti Connect Secure 0 < 9.1R18.9",
```
Comment (justmurphy) [inline csaf_files/IT/white/2024/va-24-317-01.json:141]: Change `"name"` field to reflect above change with version range.

File: csaf_files/IT/white/2024/va-24-317-01.json [hunk 9]
```diff
@@ -0,0 +1,898 @@
...
+                "product": {
+                  "name": "Ivanti Connect Secure 9.1R18.9",
+                  "product_id": "CSAFPID-0010"
+                }
+              },
+              {
+                "category": "product_version_range",
+                "name": "0 < 22.7R2.2",
+                "product": {
+                  "name": "Ivanti Connect Secure 0 < 22.7R2.2",
```
Comment (justmurphy) [inline csaf_files/IT/white/2024/va-24-317-01.json:157]: Change `"name"` field to reflect above change with version range.

File: csaf_files/IT/white/2024/va-24-317-01.json [hunk 10]
```diff
@@ -0,0 +1,898 @@
...
+                "category": "product_version",
+                "name": "9.1R18.9",
+                "product": {
+                  "name": "Ivanti Connect Secure 9.1R18.9",
+                  "product_id": "CSAFPID-0010"
+                }
+              },
+              {
+                "category": "product_version_range",
+                "name": "0 < 22.7R2.2",
```
Comment (justmurphy) [inline csaf_files/IT/white/2024/va-24-317-01.json]: This should be `"<22.7R2.2"`.

See documentation: https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#31232-branches-type---name-under-product-version-range

File: csaf_files/IT/white/2024/va-24-317-01.json [hunk 11]
```diff
@@ -0,0 +1,898 @@
...
+              }
+            ]
+          },
+          {
+            "category": "product_name",
+            "name": "Policy Secure",
+            "branches": [
+              {
+                "category": "product_version_range",
+                "name": "0 < 22.7R1",
```
Comment (justmurphy) [inline csaf_files/IT/white/2024/va-24-317-01.json]: This should be `"<22.7R1"`.

See documentation: https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#31232-branches-type---name-under-product-version-range

File: csaf_files/IT/white/2024/va-24-317-01.json [hunk 12]
```diff
@@ -0,0 +1,898 @@
...
+                "product": {
+                  "name": "Ivanti Policy Secure 0 < 22.7R1",
```
Comment (justmurphy) [inline csaf_files/IT/white/2024/va-24-317-01.json:179]: Change `"name"` field to reflect above change with version range.

File: csaf_files/IT/white/2024/va-24-317-01.json [hunk 13]
```diff
@@ -0,0 +1,898 @@
...
+                "category": "product_version",
+                "name": "22.7R1",
+                "product": {
+                  "name": "Ivanti Policy Secure 22.7R1",
+                  "product_id": "CSAFPID-0014"
+                }
+              },
+              {
+                "category": "product_version_range",
+                "name": "0 < 22.7R1.1",
```
Comment (justmurphy) [inline csaf_files/IT/white/2024/va-24-317-01.json]: This should be `"<22.7R1.1"`.

See documentation: https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#31232-branches-type---name-under-product-version-range

File: csaf_files/IT/white/2024/va-24-317-01.json [hunk 14]
```diff
@@ -0,0 +1,898 @@
...
+                "product": {
+                  "name": "Ivanti Policy Secure 0 < 22.7R1.1",
```
Comment (justmurphy) [inline csaf_files/IT/white/2024/va-24-317-01.json:195]: Change `"name"` field to reflect above change with version range.

File: csaf_files/IT/white/2024/va-24-317-01.json [hunk 15]
```diff
@@ -0,0 +1,898 @@
...
+                "category": "product_version",
+                "name": "22.7R1.1",
+                "product": {
+                  "name": "Ivanti Policy Secure 22.7R1.1",
+                  "product_id": "CSAFPID-0016"
+                }
+              },
+              {
+                "category": "product_version_range",
+                "name": "0 < 22.7R1.2",
```
Comment (justmurphy) [inline csaf_files/IT/white/2024/va-24-317-01.json]: This should be `"<22.7R1.2"`.

See documentation: https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#31232-branches-type---name-under-product-version-range

File: csaf_files/IT/white/2024/va-24-317-01.json [hunk 16]
```diff
@@ -0,0 +1,898 @@
...
+                "product": {
+                  "name": "Ivanti Policy Secure 0 < 22.7R1.2",
```
Comment (justmurphy) [inline csaf_files/IT/white/2024/va-24-317-01.json:211]: Change `"name"` field to reflect above change with version range.

**Review by justmurphy — CHANGES_REQUESTED** [review]
@amanion-cisa I have left comments re: version ranges and corresponding names for the products-ids

@mstrad would appreciate a second set of eyes on this if you have the time?

File: csaf_files/IT/white/2024/va-24-317-01.json (hunk 1 above)
Comment (amanion-cisa) [reply csaf_files/IT/white/2024/va-24-317-01.json]: Fixed in https://github.com/cisagov/CSAF/pull/134/commits/4cc4828ff6a063b14e12da35d64e7c57001938b1

File: csaf_files/IT/white/2024/va-24-317-01.json (hunk 2 above)
Comment (amanion-cisa) [reply csaf_files/IT/white/2024/va-24-317-01.json:93]: Fixed in https://github.com/cisagov/CSAF/pull/134/commits/4cc4828ff6a063b14e12da35d64e7c57001938b1

File: csaf_files/IT/white/2024/va-24-317-01.json (hunk 4 above)
Comment (amanion-cisa) [reply csaf_files/IT/white/2024/va-24-317-01.json:109]: Fixed in https://github.com/cisagov/CSAF/pull/134/commits/4cc4828ff6a063b14e12da35d64e7c57001938b1

File: csaf_files/IT/white/2024/va-24-317-01.json (hunk 5 above)
Comment (amanion-cisa) [reply csaf_files/IT/white/2024/va-24-317-01.json]: Fixed in https://github.com/cisagov/CSAF/pull/134/commits/4cc4828ff6a063b14e12da35d64e7c57001938b1

File: csaf_files/IT/white/2024/va-24-317-01.json (hunk 15 above)
Comment (amanion-cisa) [reply csaf_files/IT/white/2024/va-24-317-01.json]: Fixed in https://github.com/cisagov/CSAF/pull/134/commits/4cc4828ff6a063b14e12da35d64e7c57001938b1

File: csaf_files/IT/white/2024/va-24-317-01.json (hunk 16 above)
Comment (amanion-cisa) [reply csaf_files/IT/white/2024/va-24-317-01.json:211]: Fixed in https://github.com/cisagov/CSAF/pull/134/commits/4cc4828ff6a063b14e12da35d64e7c57001938b1

File: csaf_files/IT/white/2024/va-24-317-01.json (hunk 12 above)
Comment (amanion-cisa) [reply csaf_files/IT/white/2024/va-24-317-01.json:179]: Fixed in https://github.com/cisagov/CSAF/pull/134/commits/4cc4828ff6a063b14e12da35d64e7c57001938b1

File: csaf_files/IT/white/2024/va-24-317-01.json (hunk 14 above)
Comment (amanion-cisa) [reply csaf_files/IT/white/2024/va-24-317-01.json:195]: Fixed in https://github.com/cisagov/CSAF/pull/134/commits/4cc4828ff6a063b14e12da35d64e7c57001938b1

File: csaf_files/IT/white/2024/va-24-317-01.json (hunk 13 above)
Comment (amanion-cisa) [reply csaf_files/IT/white/2024/va-24-317-01.json]: Fixed in https://github.com/cisagov/CSAF/pull/134/commits/4cc4828ff6a063b14e12da35d64e7c57001938b1

File: csaf_files/IT/white/2024/va-24-317-01.json (hunk 3 above)
Comment (amanion-cisa) [reply csaf_files/IT/white/2024/va-24-317-01.json]: Fixed in https://github.com/cisagov/CSAF/pull/134/commits/4cc4828ff6a063b14e12da35d64e7c57001938b1

File: csaf_files/IT/white/2024/va-24-317-01.json (hunk 6 above)
Comment (amanion-cisa) [reply csaf_files/IT/white/2024/va-24-317-01.json:125]: Fixed in https://github.com/cisagov/CSAF/pull/134/commits/4cc4828ff6a063b14e12da35d64e7c57001938b1

**Review by justmurphy — APPROVED** [review]
lgtm

**Pull request merged**